src/app/view/perspectives/smartdoc/bottom-panel/smartdoc-bottom-panel.component.html

```diff
@@ -16,7 +16,7 @@
           <div hidden>
             <span #perspectiveName
                   i18n="@@smartdoc.button.perspective.switch">
-              Switch to {perspective, select, postit {post-it} search {search} table {table} smartdoc {smart document}}
+              Switch to {perspective, select, postit {post-it} search {search} table {table} document {smart document}}
```

Comment:
We use `smartdoc` string in [Perspective](https://github.com/Lumeer/web-ui/blob/devel/src/app/view/perspectives/perspective.ts#L23) enum.